pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.607</version>
```

Comment:
1.607 was the earliest I could find that supported  jenkins.model.queue.AsynchronousExecution -- I've gone ahead and bumped to 1.609 (and re-run tests) so it's tied to an LTS.